packages/combobox/src/Combobox.tsx

```diff
@@ -1351,6 +1356,51 @@ export default function Combobox<M extends boolean>({
     if (comboboxRef.current?.contains(element)) return ComboboxElement.Combobox;
   }
 }
+
+/* eslint-disable react/forbid-prop-types */
+Combobox.propTypes = {
+  // Multiselect props
+  multiselect: PropTypes.bool,
+  value: PropTypes.oneOf([
+    PropTypes.string,
+    PropTypes.arrayOf(PropTypes.string),
+  ]),
+  initialValue: PropTypes.oneOf([
+    PropTypes.string,
+    PropTypes.arrayOf(PropTypes.string),
+  ]),
+  overflow: PropTypes.oneOf(Object.values(Overflow)),
+
+  // Standard Props
+  onChange: PropTypes.func,
+  'aria-label': PropTypes.string,
+  label: PropTypes.string,
+  chipCharacterLimit: PropTypes.number,
+  chipTruncationLocation: PropTypes.oneOf(Object.values(TruncationLocation)),
+  onClear: PropTypes.func,
+  onFilter: PropTypes.func,
+  clearable: PropTypes.bool,
+  searchLoadingMessage: PropTypes.string,
+  searchErrorMessage: PropTypes.string,
+  searchEmptyMessage: PropTypes.string,
+  searchState: PropTypes.oneOf(Object.values(SearchState)),
+  errorMessage: PropTypes.string,
+  state: PropTypes.oneOf(Object.values(State)),
+  darkMode: PropTypes.bool,
+  size: PropTypes.oneOf(Object.values(ComboboxSize)),
+  disabled: PropTypes.bool,
+  description: PropTypes.string,
+  children: PropTypes.node,
+  placeholder: PropTypes.string,
+  filteredOptions: PropTypes.arrayOf(PropTypes.string),
+  // Popover Props
+  popoverZIndex: PropTypes.number,
+  usePortal: PropTypes.bool,
+  scrollContainer: PropTypes.any,
```

Comment:
Should we use `PropTypes.elementType`? Its what i remember you using in (LeafyGreenContext)[https://github.com/mongodb/leafygreen-ui/blob/main/packages/leafygreen-provider/src/LeafyGreenContext.tsx#L43]

Reply:
Yeah that makes more sense 👍 I'll make that update